### PR TITLE
Improve production tests

### LINKED ISSUE #554: Review HTML / JS / CSS modifications as close as possible from production state
Discussion started from here #542

Goal: Find a way to test HTML /JS / CSS modifications as close as possible from the production state.

### PR DESCRIPTION
**Pull request by fabienrohrer**
Attempt to fix #554

Aim: Change doc.php to use the JS and CSS ressources host on GitHub.

### CODE REVIEW

**Comment by fabienrohrer** [comment]
Demo: https://cyberbotics.com/docphp/guide/webots-devices?version=enhancement-production-test

**Comment by fabienrohrer** [comment]
As expected, it is not possible to use Github user content to access JS and CSS. I used [rawgit CDN](https://rawgit.com/) to do so, which is aimed to be used in production.

I'm ready for an intermediate review. When approved, I will drop any reference to `doctest.php` and remove the background tests in `webots.js` and `webots-doc.css`.
After a second acceptance, I will apply this patch on the FTP.

👍

File: doc.php [hunk 1]
```diff
@@ -35,26 +35,28 @@
   }
   if (!isset($repository))
     $repository = 'omichel';
+  $url = "https://raw.githubusercontent.com/$repository/webots-doc/";
+  $rawgiturl = "https://cdn.rawgit.com/$repository/webots-doc/";
```
Comment (fabienrohrer) [inline doc.php]: We could use the CDN only for the master branch, in order to increase the update time.

**Comment by omichel** [comment]
:eyes:

File: doc.php [hunk 2]
```diff
@@ -35,26 +35,28 @@
   }
   if (!isset($repository))
     $repository = 'omichel';
+  $url = "https://raw.githubusercontent.com/$repository/webots-doc/";
```
Comment (omichel) [inline doc.php]: I wouldn't define a variable like this (used only once, defined far away from its usage) and revert to the original version.

File: css/webots-doc.css [hunk 3]
```diff
@@ -195,7 +195,7 @@
 }
 .webots-doc .anchor-link-image {
   padding-left: 20px;
-  background: url(images/anchor.png) left center no-repeat;
+  background: url('./images/anchor.png') left center no-repeat;
```
Comment (omichel) [inline css/webots-doc.css]: That doesn't look simpler...

Comment (omichel) [reply css/webots-doc.css]: Do we really need this `./` prefix?

**Comment by fabienrohrer** [comment]
👍

**Comment by fabienrohrer** [comment]
👍 
(when approved, I will push these modifications on the FTP).

**Comment by fabienrohrer** [comment]
Thank you for the commit.
I push this on the FTP and test.

**Comment by fabienrohrer** [comment]
I had to fix a small issue, but now it's fixed and deployed.
I'm merging this PR. Please tell me if you find any issue.

**Comment by fabienrohrer** [comment]
So from now, the production tests will be much easier, we can test CSS and JS modifications in production, just using the `version` GET option.

**Comment by omichel** [comment]
Very good.

**Pull request merged**